CMakeLists.txt

```diff
@@ -12,22 +14,24 @@ add_subdirectory(C/plugins/storage/postgres)
 add_subdirectory(C/plugins/storage/sqlite)
 add_subdirectory(C/plugins/storage/sqlitememory)
 add_subdirectory(C/services/south)
-add_subdirectory(C/plugins/south/dummy)
-find_library(WiringPi wiringPi)
-if (WiringPi)
-	add_subdirectory(C/plugins/south/ina219)
-	add_subdirectory(C/plugins/south/dht11)
-endif()
-find_library(Modbus modbus)
-if (Modbus)
-	add_subdirectory(C/plugins/south/modbus)
-endif()
 add_subdirectory(C/tasks/north)
-add_subdirectory(C/plugins/north/omf)
-add_subdirectory(C/plugins/north/ThingSpeak)
-add_subdirectory(C/plugins/north/http-north)
+if (BUILD_C_PLUGINS)
+	add_subdirectory(C/plugins/south/dummy)
+	find_library(WiringPi wiringPi)
+	if (WiringPi)
+		add_subdirectory(C/plugins/south/ina219)
+		add_subdirectory(C/plugins/south/dht11)
+	endif()
+	find_library(Modbus modbus)
+	if (Modbus)
+		add_subdirectory(C/plugins/south/modbus)
+	endif()
+	add_subdirectory(C/plugins/north/omf)
+	add_subdirectory(C/plugins/north/ThingSpeak)
+	add_subdirectory(C/plugins/north/http-north)
+	add_subdirectory(C/plugins/filter/scale)
+endif (BUILD_C_PLUGINS)
 add_subdirectory(C/plugins/utils)
-add_subdirectory(C/plugins/filter/scale)
 # Add Python 2.7 filter
```

Comment:
C/plugins/filter/python27 is a C++ filter which embeds Python 2.7 intepreter

It should be moved into into the BUILD_C_PLUGINS conditional build as well

Reply:
This doesn't matter as filter plugins are not seen by the Python code.

Reply:
We ca move out add_subdirectory(C/plugins/filter/scale) then